src/momento/internal/aio/_scs_data_client.py

```diff
@@ -141,13 +126,17 @@ class _ScsDataClient:
     __UNSUPPORTED_DICTIONARY_FIELDS_TYPE_MSG = "Unsupported type for fields: "
     __UNSUPPORTED_DICTIONARY_ITEMS_TYPE_MSG = "Unsupported type for items: "
 
+    _DEFAULT_DEADLINE = timedelta(seconds=5)
+
     def __init__(self, configuration: Configuration, credential_provider: CredentialProvider, default_ttl: timedelta):
         endpoint = credential_provider.cache_endpoint
         self._logger = logs.logger
         self._logger.debug("Simple cache data client instantiated with endpoint: %s", endpoint)
         self._endpoint = endpoint
 
-        default_deadline: timedelta = get_default_client_deadline(configuration)
+        default_deadline: timedelta = (
+            configuration.get_transport_strategy().get_grpc_configuration().get_deadline() or self._DEFAULT_DEADLINE
+        )
```

Comment:
The deadline is not nullable in the config so this alternative shouldn't be necessary